spec/support/request/web_helper.rb

```diff
@@ -127,6 +127,12 @@ def close_select2
     page.find(:css, 'body').click
   end
 
+  def tomselect_search_and_select(value, options)
+    page.find("[name='#{options[:from]}']").sibling(".ts-wrapper").click
+    page.find(:css, '.ts-dropdown input.dropdown-input').set(value)
+    page.find(:css, '.ts-dropdown .ts-dropdown-content .option', text: value).click
```

Comment:
So what if a user doesn't click but uses the keyboard or some other device to choose a value?

Reply:
`tomselect_search_and_select` is not here to test `tom-select` but to create a helper method that search and select from a tom-select component via Ajax request:

![2022-05-10 10 23 02](https://user-images.githubusercontent.com/296452/167583495-d32e3aaf-3686-4a74-a320-1006d4f243bb.gif)